config/rbac/role.yaml

```diff
@@ -270,6 +270,25 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - quota.codeflare.dev
```

Comment:
This should also be added to the following manifest for the time being: https://github.com/project-codeflare/codeflare-operator/blob/2a4f5b8a6c5dc4dcf22be231999169eab87e08b7/config/internal/mcad/clusterrole_mcad-controller.yaml.tmpl#L14.